editor/gui/editor_file_dialog.cpp

```diff
@@ -858,8 +882,10 @@ void EditorFileDialog::update_file_list() {
 		d["path"] = cdir.path_join(dir_name);
 		d["dir"] = true;
 
+		Color folder_icon_color = get_dir_icon_color(String(d["path"]));
```

Comment:
I think you could put it directly in the method call, as the variable is used only once.